### PR TITLE
IGAPP-1179: Poi categories native

### PR DESCRIPTION
**Pull request by f1sh1918**
Show Poi Category on native app

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Nice! Tested in emulated Android and real iOS. On selecting one of the POIs, the icon reverts to the general one again by the way, is that the expected behaviour?

File: release-notes/unreleased/IGAPP-1179-poi-categories-native.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-1179
+show_in_stores: true
+platforms:
+  - android
+  - ios
+en: Categories of Location will be shown.
```
Comment (SteffiStuffel) [inline release-notes/unreleased/IGAPP-1179-poi-categories-native.yaml:6]: "of **l**ocation**s**"

**Review by SteffiStuffel — CHANGES_REQUESTED** [review]
Code looks good, but getting visual bugs, see attached screenshots 
Navigating to Berlin, everything looks fine:
not yet selected joan miro
Clicking on Joan Miro, berlin icon is just a black circle
selected joan miro - berlin black
Selecting Berlin, Joan Miro icon is dark
selected berlin again - joan miro dark

Also i get a lot of ` Mapbox [info] Request failed due to a permanent error: Canceled  {"level": "warning", "message": "Request failed due to a permanent error: Canceled ", "tag": "Mbgl-HttpRequest"}` errors in the console.

**Comment by f1sh1918** [comment]
> Code looks good, but getting visual bugs, see attached screenshots Navigating to Berlin, everything looks fine: not yet selected joan miro Clicking on Joan Miro, berlin icon is just a black circle selected joan miro - berlin black Selecting Berlin, Joan Miro icon is dark selected berlin again - joan miro dark
> 
> Also i get a lot of ` Mapbox [info] Request failed due to a permanent error: Canceled {"level": "warning", "message": "Request failed due to a permanent error: Canceled ", "tag": "Mbgl-HttpRequest"}` errors in the console.

@SteffiStuffel these look like common emulator bugs on android. could you please test (and maybe confirm) this issue on real device. We receive this warning error in all devices even if it works. Is a maplibre issues (happens f.e. also in entitlementcard)

**Review by SteffiStuffel — APPROVED** [review]

**Pull request merged**